### PR TITLE
Add vulns NSWG-ECO 443 to 447

### PR DESCRIPTION
**Pull request by vdeturckheim**

### CODE REVIEW

**Comment by lirantal** [comment]
@vdeturckheim missing `cvss_score` field

**Comment by vdeturckheim** [comment]
@ChALkeR fixed, sorry about this one

File: README.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ You can show your users you take security matters seriously and drive higher con
 
 # Code of Conduct
 
+
```
Comment (ChALkeR) [inline README.md]: Unrelated change.

File: package-lock.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
       "resolved": "https://registry.npmjs.org/topo/-/topo-3.0.0.tgz",
       "integrity": "sha512-Tlu1fGlR90iCdIPURqPiufqAlCZYzLjHYVVbcFWDMcX7+tK8hdZWAfsMrD/pBul9jqHHwFjNdf1WaxA9vTRRhw==",
       "requires": {
-        "hoek": "5.0.3"
+        "hoek": "5.x.x"
```
Comment (ChALkeR) [inline package-lock.json]: Changes in this file are also unrelated. Could those be placed in a separate PR if we need them?

**Comment by lirantal** [comment]
@vdeturckheim build fails due to `references` field needs to be an array from recent change we've made to the vuln db format.
